src/plugins/intel_cpu/src/nodes/rnn.cpp

```diff
@@ -255,25 +256,57 @@ bool RNN::isSupportedOperation(const std::shared_ptr<const ov::Node>& op, std::s
         }
 
         auto rnnCellBase = ov::as_type_ptr<const ov::op::util::RNNCellBase>(op);
-        if (rnnCellBase && rnnCellBase->get_clip() != 0.f) {
-            errorMessage = "Clipping is not supported for RNN primitive.";
-            return false;
+        if (rnnCellBase) {
+            if (rnnCellBase->get_clip() != 0.f) {
+                errorMessage = "Clipping is not supported for RNN primitive.";
+                return false;
+            }
+            if (one_of(rnnCellBase->get_type_info(),
+                    ov::op::v0::LSTMCell::get_type_info_static(),
+                    ov::op::v4::LSTMCell::get_type_info_static(),
+                    ov::op::v5::LSTMSequence::get_type_info_static()) &&
+                    rnnCellBase->get_activations() != std::vector<std::string>{"sigmoid", "tanh", "tanh"}) {
+                errorMessage = "Not supported activation functions";
+                return false;
+            } else if (rnnCellBase->get_activations() != std::vector<std::string>{"sigmoid", "tanh"}) {
+                errorMessage = "Not supported activation functions";
+                return false;
+            }
         }
 
         ov::op::RecurrentSequenceDirection direction = ov::op::RecurrentSequenceDirection::FORWARD;
-        if (ov::is_type<ov::op::v5::GRUSequence>(op)) {
-            direction = ov::as_type_ptr<const ov::op::v5::GRUSequence>(op)->get_direction();
-        } else if (ov::is_type<ov::op::v0::LSTMSequence>(op)) {
-            direction = ov::as_type_ptr<const ov::op::v0::LSTMSequence>(op)->get_direction();
-        } else if (ov::is_type<ov::op::v5::LSTMSequence>(op)) {
-            direction = ov::as_type_ptr<const ov::op::v5::LSTMSequence>(op)->get_direction();
-        } else if (ov::is_type<ov::op::v5::RNNSequence>(op)) {
-            direction = ov::as_type_ptr<const ov::op::v5::RNNSequence>(op)->get_direction();
+        size_t seqLenIdx = 2;
+        if (auto gru_seq = ov::as_type_ptr<const ov::op::v5::GRUSequence>(op)) {
+            direction = gru_seq->get_direction();
+        } else if (auto lstm_seq = ov::as_type_ptr<const ov::op::v0::LSTMSequence>(op)) {
+            if (lstm_seq->get_activations() != std::vector<std::string>{"sigmoid", "tanh", "tanh"}) {
```

Comment:
Isn't it check at line 2687

Reply:
Line 268 checks for v5::LSTMSequence. Here we do that for v0::LSTMSequence.